n3fit/src/n3fit/hyper_optimization/mongofiletrials.py

```diff
@@ -0,0 +1,250 @@
+"""
+    Hyperopt trial object for parallel hyperoptimization with MongoDB.
+    Data are fetched from MongoDB databases and stored in the form of json files within the nnfit folder
+"""
+import json
+import logging
+import os
+import subprocess
+
+from bson import SON, ObjectId
```

Comment:
`bson` should also be included in the dependencies.

Reply:
Hi @Radonirinaunimi. Thanks! this `bson` package comes with`pymongo`. We already have `pymongo` in the dependencies.